web/web.js

```diff
@@ -1,10 +1,34 @@
+import { user_info_wos } from "db";
```

Comment:
I think you missed the implementation of this in your commit. I'm not seeing it on the diff. But I am seeing `user_info`.

Reply:
Yep completely forgot to include that. Should be fixed.